### PR TITLE
Ensure compatibility with history@5 #111

### PR DESCRIPTION
**Pull request by marionebl**

### CODE REVIEW

File: src/__tests__/unit/controllers/router-store/test.tsx [hunk 1]
```diff
@@ -243,6 +244,98 @@ describe('SPA Router store', () => {
     });
   });
 
+  describe.only('listening for real history changes on history@5', () => {
```
Comment (albertogasparin) [inline src/__tests__/unit/controllers/router-store/test.tsx:247]: Please remove `.only`

**Comment by albertogasparin** [comment]
@liamqma This looks good to me, but do you think we need more changes?

File: src/common/utils/create-location/parse-path.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export function parsePath(path: string) {
+  const url = new URL(path, 'ws://a.a');
```
Comment (albertogasparin) [inline src/common/utils/create-location/parse-path.ts:2]: Out of curiosity, why `ws://a.a` ?

**Review by albertogasparin — APPROVED** [review]

File: src/common/utils/create-location/parse-path.ts (hunk 2 above)
Comment (marionebl) [reply src/common/utils/create-location/parse-path.ts:2]: Its the shortest url I could come up with 😅

File: package.json [hunk 3]
```diff
@@ -131,6 +132,7 @@
     "webpack-dev-server": "^3.11.0"
   },
   "peerDependencies": {
+    "history": "^4 || ^5",
```
[the remark this answers is not in the document]
Comment (marionebl) [reply package.json:135]: Yes, leaning on https://semver.npmjs.com/ I think they are equivalent:

Screen Shot 2022-02-15 at 10 00 31 am
Screen Shot 2022-02-15 at 10 00 19 am

**Pull request merged**